app/src/main/java/np/com/naxa/factsnepal/notification/NotificationCount.java

```diff
@@ -0,0 +1,113 @@
+package np.com.naxa.factsnepal.notification;
+
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import com.google.gson.Gson;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.lang.reflect.Type;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import io.reactivex.Observable;
+import np.com.naxa.factsnepal.utils.SharedPreferenceUtils;
+
+public class NotificationCount {
+
+    private static final String TAG = "NotificationCount";
+    public static final String KEY_NOTIFICATION_LIST = "notification_list";
+
+    SharedPreferenceUtils sharedPreferenceUtils;
+
+    Gson gson = new Gson();
+
+    int unreadCount = 0;
+
+    public NotificationCount(Context context) {
+        sharedPreferenceUtils = new SharedPreferenceUtils(context);
+    }
+
+    public JSONArray getJsonArray() throws JSONException {
+        Gson gson = new Gson();
+        ArrayList<FactsNotification> factsNotificationArrayList = FactsNotification.getDemoItems();
+        String jsonString = gson.toJson(factsNotificationArrayList);
+
+        JSONArray factsNotificationJSONArray = new JSONArray(jsonString);
+        Log.d(TAG, "getJsonArray: default" + jsonString);
+        return factsNotificationJSONArray;
+    }
+
+    long count;
+
+    public synchronized long getNotificationCount() throws JSONException {
+        count = 0;
+        if (sharedPreferenceUtils.getStringValue(KEY_NOTIFICATION_LIST, "").equals("")) {
+            return count;
```

Comment:
Remove this if condition. If the saved data is empty `count` will be 0 because it will not go through the loop.